api/src/main/java/org/apache/iceberg/Metrics.java

```diff
@@ -103,6 +134,15 @@ public Long recordCount() {
     return nullValueCounts;
   }
 
+  /**
+   * Get the number of NaN values for all fields in a file.
```

Comment:
For all `float` and `double` fields. Other entries should be omitted because they are 0.